NEWS

```diff
@@ -11,6 +11,241 @@
+#### UCM
+ * Improved log on memory remap failure
+ * Added memory hooks for CUDA memory mapping functions
+#### UCP
+ * Added ACK overhead for ZCOPY protocols event if fast completion flag is set
+ * Improved log with explicit prompt for address v2 configuration on failure
+ * Disabled global VA memory region when error handling is in use
+ * Added interface attribute flag to detect inter-node capability
+ * Enabled using CUDA staging buffers for pipeline protocols by default.
+ * Improved debuggability for remote key size computation
+ * Improve multi-send performance factors computations
+ * Improved rndv/am protocol threshold calculation.
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled marking memory domains as non-cacheable (GDR case)
+ * Improved message upon wireup failure
+ * Added new protocol infrastructure
+ * Added user_data parameter to ucp_ep_query
+ * Added global VA infrastructure and UCP/MM: Add global VA MR
+ * Improved performance nodes naming to be more informative
+ * Added enforcing same running thread when using single thread mode
+ * Added rcache overhead for protocol performance estimation from environment variable
+ * Improved protocol selection by aggregating multiple performance segments
+ * Added UCP_MEM_MAP_BLOCK API flag to enforce blocking memory mapping
+ * Allowed up-to 64 lanes for systems with many transports or devices
+ * Added rendezvous performance variants
+ * Improved lane intersection computation for lane reuse
+ * Added protocol performance data structure
+ * Removed redundant fields from request structure
+ * Added usage tracker to worker
+ * Added protocol probing for rendezvous for variants support
+#### UCS
+ * Improve non-temporal data transfer implementation
+ * Allowed configuration with wildcard prefix and name matching
+ * Defined a non-inline function to allow ASAN validation
+ * Reduced stack size in topology detection code
+ * Improved bitmaps configuration parsing with wider bitfield
+ * Added options to set topology distance between devices
+ * Improved log level for backtrace dump
+ * Improve VFS unix socket watch by using user private folder
+ * Added general subnet matching support infrastructure
+ * Allowed user-based existing data copy when appending to array
+ * Improved time units description
+ * Added x86 non-temporal buffer transfers to skip cache
+#### UCT
+ * Improved API documentation for memory domain flags
+ * Improved GPUDirect detection support
+### Bug
+#### AZP
+ * Fixed CUDA-related segfaults by adding host IPC capability to docker
+ * Fixed CI failures by using different ports in one wire-compat job
+ * Fixed various wire-compat test issue
+ * Cleaned up git fetch command usage
+ * Disabled wire-compat tests on non applicable machines
+#### Build
+ * Fixed using correct ASAN version for running tests
+ * Fixed git ignore file by added various test programs
+ * Clean-up duplicated field in formatting file
```

Comment:
do we need to mention? or maybe rephrase

Reply:
agree removed